graylog2-server/src/main/java/org/graylog2/filters/ExtractorFilter.java

```diff
@@ -77,8 +79,14 @@ public boolean filter(Message msg) {
                 extractor.runExtractor(msg);
             } catch (Exception e) {
                 extractor.incrementExceptions();
-                LOG.error("Could not apply extractor \"" + extractor.getTitle() + "\" (id=" + extractor.getId() + ") "
-                        + "to message " + msg.getId(), e);
+                final String error = "Could not apply extractor <" + extractor.getTitle() + "(" + extractor.getId() + ")>";
```

Comment:
Please prove the code with unit tests

Reply:
Done. Don't you love it if a test is 4 times as much complicated as the actual code? :see_no_evil: